packages/wrangler/pages/functions/template-worker.ts

```diff
@@ -88,9 +88,11 @@ function* executeRequest(request: Request, env: Env) {
     }
   }
 
-  // Finally, yield to the asset-server
```

Comment:
I can't add a suggestion on a non-edited line, but on line 45, replace `env` with `_env` to remove the lint warning.

Reply:
I'd rather not, since we swap it out for the conventional `env.SERVICE` in build. I'll just suppress the warning.

Reply:
ah right. ok.